packages/Spellcheck-Tests.package/ParserTests.class/instance/testParsingWithSeparator.st

```diff
@@ -7,6 +7,8 @@ testParsingWithSeparator
 	parser := SpellingTextParser new.
 	strings := parser parseText: text.
 	
+	self assert: (strings size = 4).
 	self assert: ((text copyFrom: 1 to: (strings at: 1)) = 'Hallo').
-	self assert: ((text copyFrom: 10 to: (strings at: 10)) = 'Max.xx;;').
+	self assert: ((text copyFrom: 10 to: (strings at: 10)) = 'Max').
+	self assert: ((text copyFrom: 14 to: (strings at: 14)) = 'xx').
```

Comment:
Why is 'xx' a valid word?

Reply:
It is not a valid word, it is just a word. The test has nothing to do with spell-checking, just parsing words.